README.md

```diff
@@ -22,6 +23,21 @@ Table Of Contents
   * [License](#license)
 
 
+How to edit navigation data
+---------------------------
+
+The navigation data is written in JSON and can be edited via the Github edit tool.
+
+Steps to edit navigation data:
+
+- Open the navigation data file in the [Github edit tool](https://github.com/Financial-Times/origami-navigation-service-data/edit/master/data/navigation.json)
+- Make the necessary changes
```

Comment:
This assumes that the people editing the file are people that know what "the necessary changes" should be and should look like?